LayoutTests/TestExpectations

```diff
@@ -6864,14 +6864,16 @@ imported/w3c/web-platform-tests/scroll-animations/view-timelines/view-timeline-s
 imported/w3c/web-platform-tests/scroll-animations/view-timelines/view-timeline-sticky-inline.html [ Skip ]
 imported/w3c/web-platform-tests/scroll-animations/view-timelines/view-timeline-subject-size-changes.html [ Skip ]
 
-# webkit.org/b/229520 CSS @scope unimplemented
-imported/w3c/web-platform-tests/css/css-cascade/import-conditional-002.html [ ImageOnlyFailure ]
-imported/w3c/web-platform-tests/css/css-cascade/scope-visited.html [ ImageOnlyFailure ]
-
 # other scroll-animations failures
 webkit.org/b/263871 imported/w3c/web-platform-tests/scroll-animations/css/animation-timeline-none.html [ Failure ]
 webkit.org/b/263872 imported/w3c/web-platform-tests/scroll-animations/css/printing/animation-timeline-none-with-progress-print.tentative.html [ ImageOnlyFailure ]
 webkit.org/b/263873 imported/w3c/web-platform-tests/scroll-animations/css/printing/scroll-timeline-default-iframe-print.html [ ImageOnlyFailure ]
+webkit.org/b/263873 imported/w3c/web-platform-tests/scroll-animations/css/progress-based-animation-timeline.html [ Pass Failure ]
+webkit.org/b/263873 imported/w3c/web-platform-tests/scroll-animations/css/view-timeline-dynamic.html [ Pass Failure ]
```

Comment:
Indeed, I copied the wrong bug, I tied them with [bug 263871](https://webkit.org/b/263871). The tests log an incorrect value which changes from run to run probably because of animation sampling. I didn't look too hard since it's a failure we'll fix down the line when we get to implementing scroll-driven animations proper.